skbio/sequence/tests/test_distance.py

```diff
@@ -126,5 +126,76 @@ def test_sequences_with_metadata(self):
             self.assertEqual(distance, 0.75)
 
 
+class TestKmerDistance(unittest.TestCase):
+    def test_distance_default_kwargs(self):
```

Comment:
These tests all start with `test_distance_*`; since they are in a class solely responsible for testing `kmer_distance` I'd remove `_distance_` from the names since it's redundant.